src/Execution/Arguments/ArgumentSet.php

```diff
@@ -153,8 +153,11 @@ protected static function applyFieldBuilderDirectives(self $argumentSet, object
     {
         $argumentSet->directives
             ->filter(Utils::instanceofMatcher(FieldBuilderDirective::class))
-            ->each(static function (FieldBuilderDirective $fieldBuilderDirective) use (&$builder): void {
-                $builder = $fieldBuilderDirective->handleFieldBuilder($builder);
+            ->each(static function (FieldBuilderDirective $fieldBuilderDirective) use (&$builder, $argumentSet): void {
+                // @phpstan-ignore-next-line
+                $fieldBuilderDirective->argsForHandleFieldBuilder = $argumentSet->toArray();
+                // @phpstan-ignore-next-line
+                $builder = $fieldBuilderDirective->handleFieldBuilder($builder, $argumentSet->toArray());
```

Comment:
Those changes seem unrelated, please clean up.

Reply:
Oops messed how that got there from another branch :)